clone_update_repo.sh

```diff
@@ -1,79 +1,118 @@
+#!/bin/bash
+
+# Added color variables
+txtrst=$(tput sgr0) 
+txtred=$(tput setaf 1)
+
 # Add announcement that this script has just started with timestamp
-# TODO
+echo "${txtred}Timestamp:${txtrst} Script has started at $(date +%T)"
 
 # Clone the "happy_bank_core" repo from Github
-# TODO
+echo "${txtred}Cloned remote repository:${txtrst}"
+git clone https://github.com/JanMate/happy_bank_core.git
+cd happy_bank_core
 
 # Create a new branch "update-readme" and checkout it
-# TODO
+echo "${txtred}Created new branch:${txtrst}"
+git checkout -b "update-readme"
+git checkout update-readme #already on this branch
```

Comment:
I would say that  additional checkout command is not necessary because by default you are automatically moved to a new branch after git checkout -b command. I just wanted to follow the task "Create a new branch "update-readme" and checkout it"

Reply:
I see. So, if it's not necessary, get rid of it.

Reply:
There are 2 ways how to meet the requirement:

1. `git branch <branch-name>; git checkout <branch-name>`
2. `git checkout -b <branch-name>`

Since you chose the second one (simpler one), there is no need to do extra checkout.